Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -53,8 +53,8 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed?
         return unless MacOS::Xcode.outdated?
 
-        # CI images are going to end up outdated so don't complain.
-        return if ENV["TRAVIS"] || ENV["CIRCLECI"]
+        # Travis CI images are going to end up outdated so don't complain.
```

Comment:
Might be worth noting _why_ we want to limit to just Travis in case this comes up again, especially as the commit/PR/issue history [suggested](https://github.com/Homebrew/brew/pull/1576#issuecomment-263067831) ([to me at least](https://github.com/Homebrew/brew/pull/1983#issuecomment-279082630)) that other CI providers could be added upon request?

> To detail more why the `TRAVIS` check is here specifically (and why this should be reverted): this check exists because Homebrew's CI tests for this repository use Travis CI and we want to be able to run `brew doctor` without this specific thing failing. If we were building binary packages with Travis CI or Circle CI we would want this to fail as we only want to build our binary packages on the latest versions of Xcode.